main/saturn-node.js

```diff
@@ -159,6 +160,10 @@ async function start (/** @type {Context} */ ctx) {
     console.log(msg)
     appendToChildLog(msg)
     ctx.recordActivity({ source: 'Saturn', type: 'info', message: msg })
+    Sentry.captureException(new Error(msg), scope => {
+      scope.setExtra('logs', getLog())
```

Comment:
Maybe `saturnLogs` or `moduleLogs` would be a more descriptive key?

Reply:
> Maybe `saturnLogs` or `moduleLogs` would be a more descriptive key?

Do you feel strongly about this? For as long as this is the only key we attach, I find it self explanatory enough.



> On second thought: When the process exits, the stdout and stderr streams may not be fully flushed yet. It would be more robust to report the logs from the `childProcess.on('close')` listener.

Great catch! https://github.com/filecoin-station/filecoin-station/pull/280

Reply:
> Do you feel strongly about this? For as long as this is the only key we attach, I find it self explanatory enough.

Fair enough 👍🏻